src/property-filter/internal.tsx

```diff
@@ -0,0 +1,456 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useImperativeHandle, useRef, useState } from 'react';
+import clsx from 'clsx';
+
+import { PropertyFilterOperator } from '@cloudscape-design/collection-hooks';
+
+import { InternalButton } from '../button/internal';
+import { useInternalI18n } from '../i18n/context';
+import { getBaseProps } from '../internal/base-component';
+import { AutosuggestInputRef } from '../internal/components/autosuggest-input';
+import TokenList from '../internal/components/token-list';
+import { fireNonCancelableEvent } from '../internal/events';
+import { InternalBaseComponentProps } from '../internal/hooks/use-base-component';
+import { useUniqueId } from '../internal/hooks/use-unique-id/index';
+import { SomeRequired } from '../internal/types';
+import { joinStrings } from '../internal/utils/strings';
+import InternalSpaceBetween from '../space-between/internal';
+import { SearchResults } from '../text-filter/search-results';
+import { getAllowedOperators, getAutosuggestOptions, getQueryActions, parseText } from './controller';
+import { getI18nToken } from './i18n-utils';
+import {
+  ComparisonOperator,
+  ExtendedOperator,
+  FilteringProperty,
+  FormattedToken,
+  InternalFilteringOption,
+  InternalFilteringProperty,
+  InternalFreeTextFiltering,
+  InternalQuery,
+  ParsedText,
+  PropertyFilterProps,
+  Ref,
+  Token,
+} from './interfaces';
+import { PropertyEditor } from './property-editor';
+import PropertyFilterAutosuggest, { PropertyFilterAutosuggestProps } from './property-filter-autosuggest';
+import { TokenButton } from './token';
+import { useLoadItems } from './use-load-items';
+import { matchTokenValue } from './utils';
+
+import styles from './styles.css.js';
+
+export interface I18nStringsExt {
+  tokenEditorTokenActionsLabel?: (token: FormattedToken) => string;
+  tokenEditorTokenRemoveLabel?: (token: FormattedToken) => string;
+  tokenEditorTokenRemoveFromGroupLabel?: (token: FormattedToken) => string;
+  tokenEditorAddNewTokenLabel?: string;
+  tokenEditorAddTokenActionsLabel?: string;
+  tokenEditorAddExistingTokenLabel?: (token: FormattedToken) => string;
+}
+
+export type PropertyFilterInternalProps = SomeRequired<
+  PropertyFilterProps,
+  'filteringOptions' | 'customGroupsText' | 'disableFreeTextFiltering'
+> &
+  InternalBaseComponentProps & {
+    enableTokenGroups?: boolean;
+    i18nStringsExt?: I18nStringsExt;
+  };
+
+const PropertyFilterInternal = React.forwardRef(
```

Comment:
Am I supposed to review this whole file, or is there anything in particular changed after moving from `property-filter/index.ts`?

Reply:
There is nothing special but extending the internal props with `enableTokenGroups` and `i18nStringsExt`. The first prop is a flag that is to become public once we are ready to release token groups. The second prop is a set of extra properties to be added to i18nStrings.